packages/components/src/card/docs/footer.md

```diff
@@ -0,0 +1,26 @@
+# CardFooter
+
+CardFooter renders an optional footer within a [`<Card />`](../).
+
+## Usage
+
+```jsx
+import { Card, CardFooter } from '@wordpress/components';
+
+const Example = () => (
+	<Card>
+		<CardFooter>...</CardFooter>
+	</Card>
+);
+```
+
+## Props
+
+<!-- prettier-ignore -->
+Name | Type | Default | Description
+--- | --- | --- | ---
+`isShady` | `boolean` | `false` | Renders with a light gray background color.
```

Comment:
On Reakit we [extract prop types](https://github.com/reakit/reakit/blob/6f48b4eba378e7853b9583d238a4bede996e7c94/scripts/build/utils.js#L549-L622) from the [TypeScript definitions](https://github.com/reakit/reakit/blob/6f48b4eba378e7853b9583d238a4bede996e7c94/packages/reakit/src/Dialog/Dialog.tsx#L21-L77) and inject them into the README files automatically running `npm run docs`.

This may be not trivial, but I think we can use AST to parse some JSDoc in the component file?

Reply:
Those props are shared between multiple components through the context and each individual component can override them. That's why I was thinking that we could document them in one document and reference elsewhere. We are using JSDoc in Gutenberg and it's configured to work nearly the same as TypeScript. Still, the challenge is, how to document the same props only in one file :)